src/main/java/org/javademos/java21/jep456/UnnamedPatternsAndVariablesDemo.java

```diff
@@ -0,0 +1,65 @@
+package org.javademos.java21.jep456;
```

Comment:
package name should be `java22`

Unnamed patterns and variables were a preview in Java 21 with JEP 443. This is the full implementation comming in JDK 22.